docs/administration/configuration/configuring-access/index.md

```diff
@@ -24,8 +24,64 @@ Helm value accordingly.
 
 ## User registration via Single Sign-On
 
-Handling of users with Single Sign-On (SSO) is also possible. Currently,
-this is only available for CERN deployments via [`components.reana_ui.cern_sso`](https://github.com/reanahub/reana/tree/master/helm/reana)
+Handling of users with Single Sign-On (SSO) is also possible. Currently SSO is supported for CERN as well as for any third-party SSO provider which supports [Keycloak](https://www.keycloak.org/).
+
+### Keycloak Single Sign-On Configuration
+
+First of all, to integrate REANA with your Keycloak instance, you need to create a new client from the Keycloak's admin dashboard.
+There are many configuration options, but these are the minimal changes you have to make:
+
+- _Valid Redirect URIs_ should be set to `https://<your-reana-hostname>/api/oauth/authorized/keycloak/` (see Keycloak's [Access Settings](https://www.keycloak.org/docs/latest/server_admin/#access-settings))
+- _Client authentication_ should be enabled in order to get the client ID and the client secret (see Keycloak's [Confidential client credentials](https://www.keycloak.org/docs/latest/server_admin/#_client-credentials))
+
+You can then configure REANA to use your third-party SSO provider by adding an item with the needed configuration to the [`login`](https://github.com/reanahub/reana/tree/master/helm/reana) list in the Helm values:
```

Comment:
... your Keycloak provider with the following configuration of the login list in your Helm values: